plugins/searchplugin.py

```diff
@@ -105,3 +105,18 @@ def googleStuff(self, ctx, msg):
             ctx.msg("Here is the search result for '{:s}': https://www.google.com/#q={:s}".format(msg.noCmdMsg(), url), msg.replyTo)
         else:
             ctx.msg("Something has failed. Please contact Garth about it.", msg.replyTo)
+
+    @plushieCmd("youtube")
+    @commandDoc(extra="<item to search>", doc="Has Plushie search YouTube for <item to search>")
+    def youtubeStuff(self, ctx, msg):
+        args = msg.getArgs()
+
+        if len(args) < 1:
+            ctx.msg("There is nothing to search.", msg.replyTo)
+            return
+
+        if len(args) > 0:
```

Comment:
This line is superfluous. `if len(args) < 1` catches all number <1, so the only remaining numbers are naturally going to be >0.